appveyor.yml

```diff
@@ -66,11 +66,10 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase
+  - del /Q coverage.xml
```

Comment:
Hm, who creates `coverage.xml`? Is that created internally by the 'pytest-cov' plugin, and then `py.test` converts that to `unittests.xml` (in JUnit XML format)?